pkg/actors/actors.go

```diff
@@ -1175,6 +1287,27 @@ func (a *actorsRuntime) DeleteReminder(ctx context.Context, req *DeleteReminderR
 		return ErrReminderOpActorNotHosted
 	}
 
+	if a.schedulerClients != nil && a.schedulerReminders {
+		internalDeleteJobReq := &schedulerv1pb.DeleteJobRequest{
```

Comment:
Ditto, please migrate to a separate function.

But--this should have been implemented in a struct that implements the `Reminder` interface

Reply:
sure thing I can mv to a separate func, but plz see my response above as to why we didn't implement the reminder interface